tests/utils.go

```diff
@@ -301,9 +301,13 @@ func RunVMIAndExpectLaunchIgnoreWarnings(vmi *v1.VirtualMachineInstance, timeout
 }
 
 func RunVMIAndExpectScheduling(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
+	wp := watcher.WarningsPolicy{FailOnWarnings: true}
+	return RunVMIAndExpectSchedulingWithWarningPolicy(vmi, timeout, wp)
+}
+
+func RunVMIAndExpectSchedulingWithWarningPolicy(vmi *v1.VirtualMachineInstance, timeout int, wp watcher.WarningsPolicy) *v1.VirtualMachineInstance {
```

Comment:
nit: couldn't this func be un-exported ?

Reply:
Unfortunately this isn't possible:
`vmi_configuration_test.go` is part of `package tests_test`.
`utils.go` is part of `package tests`.

I have no idea why these were split into different packages. That's indeed weird.

Reply:
Thanks. I just saw they were on the same folder; should have checked the pkg as well. 

OK then.